### DISCUSSION TITLE
chat/

### DISCUSSION THREAD

**Discussion by giscus** [CloneWith/CloneWith.github.io#14] [Ideas]
# chat/

https://clonewith.github.io/chat/

**Comment by CloneWith** [comment]
想问一个问题：

我部署网页的过程中，使用 MkDocs 文档中的方法添加了缓存操作，但是运行 Action 的时候仍然报错找不到文件夹，详情请见我的仓库的 Actions 页。不知道该怎么解决这个问题？

**Reply by NaughtyChas** [reply]
感觉还是换个框架好点。MkDocs比较偏向于写文档而不是个人博客（

**Comment by deeperlearner** [comment]
頭像很可愛😋😋

**Comment by CloneWith** [comment]
🎉❤520~脱单~（不是脱单不是脱单）攻略❤🎉

👇点击回复框👇
编辑发送“咱俩试试？”💌
即有机会获得丰厚💕礼品，让你拿到手软💕

⭕兑奖日期：**即日 - 2025年5月21日 00:00**

抽奖概率公示：

| 回应 | 概率 |
| :-: | :-: |
| 真的可以嘛？！ | 1‰ |
| 喵喵喵？ | 1‰ |
| 随机静态错误 | 51.4% |
| 随机运行时错误 | 11.4% |
| "undefined" | undefined% |

更有机会抽中隐藏款！心动不如行动，快来试试试试试试试试试试试试试试

**Reply by XmasSiYuan** [reply]
咱俩试试？
